parser/options.go

```diff
@@ -4,9 +4,15 @@ import "github.com/gomarkdown/markdown/ast"
 
 // ParserOptions is a collection of supplementary parameters tweaking the behavior of various parts of the parser.
 type ParserOptions struct {
-	ParserHook BlockFunc
+	ParserHook  BlockFunc
+	IncludeHook IncludeFunc
```

Comment:
`ReadIncludeHook` or maybe  `ReadIncludeFunc` or `ReadIncludeFn` (like `walkFn` in `filepath.Walk` or `ReadInclude` would be a better name (not sure if `Hook` is the right term in this context).